setup.py

```diff
@@ -48,7 +48,10 @@ def get_requirements():
             "pubtools-pulp-maintenance-off = pubtools._pulp.tasks.set_maintenance.set_maintenance_off:entry_point",
             "pubtools-pulp-publish = pubtools._pulp.tasks.publish:entry_point",
             "pubtools-pulp-push = pubtools._pulp.tasks.push:entry_point",
-        ]
+        ],
+        "pubtools.hooks": [
```

Comment:
This part is not right, this should live in pubtools-exodus.

Reply:
ah, okay, thanks.